TCGA notebook validation plots (#64)
d3b-center/annoFuse: README.md

### Prerequisites for cohort level analysis

 - merge calls from each caller for you cohort and a column `annots` with additional annotation (eg from FusionAnnotator or caller specific annotation we have used FusionAnnotator in our vignettes)

 - reference folder  with a gene [genelistreference.txt](https://github.com/d3b-center/annoFuse/blob/master/inst/extdata/genelistreference.txt) and [fusionreference.txt](https://github.com/d3b-center/annoFuse/blob/master/inst/extdata/fusionreference.txt) inst/extdata has reference files we've used in our vignettes.

> These files were created using publically available gene lists [kinases](http://kinase.com/human/kinome/tables/Kincat_Hsap.08.02.xls), [oncogenes](http://www.bushmanlab.org/assets/doc/allOnco_Feb2017.tsv), [tumor suppressors](https://bioinfo.uth.edu/TSGene/Human_TSGs.txt?csrt=5027697123997809089), curated transcription factors [@doi:10.1016/j.cell.2018.01.029], [COSMIC genes](https://cancer.sanger.ac.uk/census), [TCGA fusions](https://tumorfusions.org/PanCanFusV2/downloads/pancanfus.txt.gz) and _MYBL1_ [@doi:10.1073/pnas.1300252110], _SNCAIP_ [@doi:10.1038/nature11327], _FOXR2_ [@doi:10.1016/j.cell.2016.01.015], _TTYH1_ [@doi:10.1038/ng.2849], and _TERT_ [@doi:10.1038/ng.3438; @doi:10.1002/gcc.22110; @doi:10.1016/j.canlet.2014.11.057; @doi:10.1007/s11910-017-0722-5] were added to the oncogene list and _BCOR_ [@doi:10.1016/j.cell.2016.01.015] and _QKI_ [@doi:10.1038/ng.3500] were added to the tumor suppressor gene list based on pediatric cancer literature review.

Reviewer: Should these citations be expanded in the full format? Because as of now the simple md rendering does not work

Reviewer 2: @kgaonkar6 yes, I think this format is specific to ManuBot

Author: Ah yes I can update those to full format citations

Reviewer: Ooh I love ManuBot, we just started using it for a collaborative project. 
I can feel Casey's long range influence 😄

Author: Does the updated format look ok? Or should I put all this information in the wiki page or something?

> The fusion reference is made up of TCGA fusions (Hu et al. 2018), gene reference file was created from publically available reference files : oncogene (http://www.bushmanlab.org/assets/doc/allOnco_Feb2017.tsv), tumor suppressor (Zhao et al. 2013)-(Zhao et al. 2016), kinase from pfam location database (http://hgdownload.soe.ucsc.edu/goldenPath/hg38/database/ucscGenePfam.txt.gz) and id description from pfamID Description database (http://hgdownload.soe.ucsc.edu/goldenPath/hg38/database/pfamDesc.txt.gz), as well as from Human Kinome database (Manning et al. 2002), transcription factor (Lambert et al. 2018), and  whether it has been reported in the Catalogue of Somatic Mutations in Cancer (COSMIC) Cancer Gene Census (Sondka et al. 2018). In addition,  we added the following genes specific to pediatric cancer from literature review; _MYBL1_ (Ramkissoon et al. 2013), _SNCAIP_ (Northcott et al. 2012), _FOXR2_ (Sturm et al. 2016), _TTYH1_ (Kleinman et al. 2014), and _TERT_ (Valentijn et al. 2015);(Lindner et al. 2015);(Karlsson et al. 2015);(Karsy et al. 2017)  were added to the oncogene list and _BCOR_ (Sturm et al. 2016) and _QKI_ (Bandopadhayay et al. 2016).

Reviewer 2: ```suggestion
The fusion reference is a compilation of:
  -  TCGA fusions (Hu et al. 2018)
  - gene reference file, which was created from publicly-available reference files
  - oncogene (http://www.bushmanlab.org/assets/doc/allOnco_Feb2017.tsv)
  - tumor suppressor (Zhao et al. 2013, Zhao et al. 2016)
  - kinase from pfam location database (http://hgdownload.soe.ucsc.edu/goldenPath/hg38/database/ucscGenePfam.txt.gz)
  - id description from pfamID Description database (http://hgdownload.soe.ucsc.edu/goldenPath/hg38/database/pfamDesc.txt.gz) and Human Kinome database (Manning et al. 2002)
  - transcription factor (Lambert et al. 2018)
  - Catalogue of Somatic Mutations in Cancer (COSMIC) Cancer Gene Census fusions (Sondka et al. 2018)
  - Oncogenes and tumor suppressor genes specific to pediatric cancer were added based on literature review; _MYBL1_ (Ramkissoon et al. 2013), _SNCAIP_ (Northcott et al. 2012), _FOXR2_ (Sturm et al. 2016), _TTYH1_ (Kleinman et al. 2014), and _TERT_ (Valentijn et al. 2015);(Lindner et al. 2015);(Karlsson et al. 2015);(Karsy et al. 2017) were added to the oncogene list. _BCOR_ (Sturm et al. 2016) and _QKI_ (Bandopadhayay et al. 2016) were added to the tumor suppressor list.

```

Reviewer 2: @kgaonkar6 these citations are good, but the papers are not linked - can you just hyperlink to the pubmed page for all of them?

> The fusion reference is made up of TCGA fusions (Hu et al. 2018), gene reference file was created from publically available reference files : oncogene (http://www.bushmanlab.org/assets/doc/allOnco_Feb2017.tsv), tumor suppressor (Zhao et al. 2013)-(Zhao et al. 2016), kinase from pfam location database (http://hgdownload.soe.ucsc.edu/goldenPath/hg38/database/ucscGenePfam.txt.gz) and id description from pfamID Description database (http://hgdownload.soe.ucsc.edu/goldenPath/hg38/database/pfamDesc.txt.gz), as well as from Human Kinome database (Manning et al. 2002), transcription factor (Lambert et al. 2018), and  whether it has been reported in the Catalogue of Somatic Mutations in Cancer (COSMIC) Cancer Gene Census (Sondka et al. 2018). In addition,  we added the following genes specific to pediatric cancer from literature review; _MYBL1_ (Ramkissoon et al. 2013), _SNCAIP_ (Northcott et al. 2012), _FOXR2_ (Sturm et al. 2016), _TTYH1_ (Kleinman et al. 2014), and _TERT_ (Valentijn et al. 2015);(Lindner et al. 2015);(Karlsson et al. 2015);(Karsy et al. 2017)  were added to the oncogene list and _BCOR_ (Sturm et al. 2016) and _QKI_ (Bandopadhayay et al. 2016).

Reviewer 2: ```suggestion
The fusion reference is a compilation of the annotations listed in the table below.

| Annotation                    | File                                                                        | Source                                                                                                                                                                                                                       |
|-------------------------------|-----------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| pfamID                        | http://hgdownload.soe.ucsc.edu/goldenPath/hg38/database/pfamDesc.txt.gz     | UCSC pfamID Description database                                                                                                                                                                                             |
| Domain Location               | http://hgdownload.soe.ucsc.edu/goldenPath/hg38/database/ucscGenePfam.txt.gz | UCSC pfamID Description database                                                                                                                                                                                             |
| TCGA fusions                  | https://tumorfusions.org/PanCanFusV2/downloads/pancanfus.txt.gz             | TumorFusions: an integrative   resource for cancer-associated transcript fusions PMID: 29099951                                                                                                                              |
| Oncogenes                     | http://www.bushmanlab.org/assets/doc/allOnco_Feb2017.tsv                    | www.bushmanlab.org                                                                                                                                                                                                           |
| Tumor suppressor genes (TSGs) | https://bioinfo.uth.edu/TSGene/Human_TSGs.txt?csrt=5027697123997809089      | Tumor Suppressor Gene Database   2.0 PMIDs: 23066107, 26590405                                                                                                                                                               |
| Kinases                       | http://kinase.com/human/kinome/tables/Kincat_Hsap.08.02.xls                 |      The protein kinase complement of the human genome PMID: 12471243                                                                                                                                                        |
| COSMIC genes                  | https://cancer.sanger.ac.uk/census                                          | Catalogue of Somatic Mutations   in Cancer                                                                                                                                                                                   |
| Pediatric-specific oncogenes  | _MYBL1, SNCAIP, FOXR2, TTYH1, TERT_                                           | doi:10.1073/pnas.1300252110,   doi:10.1038/nature11327, doi:10.1016/j.cell.2016.01.015, doi:10.1038/ng.2849,   doi:10.1038/ng.3438, doi:10.1002/gcc.22110, doi:10.1016/j.canlet.2014.11.057,   doi:10.1007/s11910-017-0722-5 |
| Pediatric-specific TSGs       | _BCOR_, _QKI_                                                                   | doi:10.1016/j.cell.2016.01.015,   doi:10.1038/ng.3500                                                                                                                                                                        |
```
